{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2256,"time":"2018-11-27T03:18:39.974Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2256,"time":"2018-11-27T03:18:40.097Z","msg":"The newrelic module must be the first module required.\nThe following modules were required before newrelic and are not being instrumented:\n\texpress: /Users/rao/Documents/hr105/TicketHogger/_rg_proxy/node_modules/express/index.js\n\tq: /Users/rao/Documents/hr105/TicketHogger/_rg_proxy/node_modules/qs/lib/index.js"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2256,"time":"2018-11-27T03:18:40.239Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2256,"time":"2018-11-27T03:18:42.054Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2256,"time":"2018-11-27T03:18:42.054Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjUzMzkyMzkxLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDEyODEwNDg2NTdd.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2273,"time":"2018-11-27T03:19:07.927Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2273,"time":"2018-11-27T03:19:08.048Z","msg":"The newrelic module must be the first module required.\nThe following modules were required before newrelic and are not being instrumented:\n\texpress: /Users/rao/Documents/hr105/TicketHogger/_rg_proxy/node_modules/express/index.js\n\tq: /Users/rao/Documents/hr105/TicketHogger/_rg_proxy/node_modules/qs/lib/index.js"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2273,"time":"2018-11-27T03:19:08.176Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2273,"time":"2018-11-27T03:19:09.915Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2273,"time":"2018-11-27T03:19:09.915Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjUzMzkzNjgxLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDI3OTc0NjcwODdd.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2291,"time":"2018-11-27T03:21:45.395Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2291,"time":"2018-11-27T03:21:45.709Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2291,"time":"2018-11-27T03:21:47.821Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2291,"time":"2018-11-27T03:21:47.821Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjUzNDAxMjY4LGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDEzODAwMDA0MTRd.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2304,"time":"2018-11-27T03:21:48.155Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2304,"time":"2018-11-27T03:21:48.352Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2304,"time":"2018-11-27T03:21:50.254Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2304,"time":"2018-11-27T03:21:50.255Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjUzNDAxMzYwLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDI4MDY3NzI3MDBd.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:21:59.987Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:22:00.182Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:22:01.255Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:22:01.255Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjUzNDAxNzMzLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDQ3NTE0MzY0NF0.","component":"collector_api"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:36:02.497Z","msg":"analytics event overflow, dropped 1073 events; try increasing your limit above 10000","component":"Harvest"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T03:37:02.497Z","msg":"analytics event overflow, dropped 778 events; try increasing your limit above 10000","component":"Harvest"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-27T08:35:49.421Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T08:35:49.530Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T08:35:49.531Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T08:35:49.549Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-27T08:35:49.549Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-27T17:05:21.888Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T17:05:21.892Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T17:05:21.893Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-27T17:05:21.893Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-27T17:05:21.893Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-28T04:40:44.871Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-28T04:40:44.884Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-28T04:40:44.888Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-28T04:40:44.889Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-28T04:40:44.889Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-29T06:36:13.309Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T06:36:13.322Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T06:36:13.324Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T06:36:13.325Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-29T06:36:13.325Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-29T16:50:49.030Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T16:50:49.030Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T16:50:49.030Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-29T16:50:49.030Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-29T16:50:49.030Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-30T07:10:08.211Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T07:10:08.213Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T07:10:08.217Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T07:10:08.217Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"collector-005.newrelic.com","pid":2319,"time":"2018-11-30T07:10:08.217Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ENOTFOUND","code":"ENOTFOUND","syscall":"getaddrinfo","host":"collector-005.newrelic.com","port":443,"stack":"Error: getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)","message":"getaddrinfo ENOTFOUND collector-005.newrelic.com collector-005.newrelic.com:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:37:02.864Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:37:02.868Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:37:02.868Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:37:02.868Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:37:02.868Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:43:02.802Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:43:02.802Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:43:02.802Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:43:02.802Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:43:02.802Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:45:02.875Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:45:02.875Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:45:02.875Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:45:02.875Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:45:02.875Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:49:02.933Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:49:02.934Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:49:02.934Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:49:02.934Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:49:02.934Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:50:03.023Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:50:03.024Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:50:03.024Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:50:03.024Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:50:03.024Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:51:03.403Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:51:03.403Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:51:03.403Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:51:03.403Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:51:03.404Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:54:03.012Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:54:03.012Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:54:03.012Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:54:03.012Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T22:54:03.012Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:00:02.945Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:00:02.945Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:00:02.945Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:00:02.945Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:00:02.946Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:01:02.886Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:01:02.886Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:01:02.886Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:01:02.887Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:01:02.887Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:02:02.921Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:02:02.921Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:02:02.921Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:02:02.921Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:02:02.921Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:08:02.965Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:08:02.965Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:08:02.965Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:08:02.965Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:08:02.965Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:14:02.996Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:14:02.996Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:14:02.996Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:14:02.997Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:14:02.997Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:19:03.025Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:19:03.025Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:19:03.025Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:19:03.025Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:19:03.026Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:23:02.995Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:23:02.996Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:23:02.996Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:23:02.996Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:23:02.996Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:27:03.080Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:27:03.080Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:27:03.080Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:27:03.081Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:27:03.081Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:28:03.159Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:28:03.159Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:28:03.159Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:28:03.159Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:28:03.159Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:39:03.329Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:39:03.329Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:39:03.329Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:39:03.329Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:39:03.329Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:54:03.150Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:54:03.150Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:54:03.150Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:54:03.150Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:54:03.151Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:58:03.147Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:58:03.147Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:58:03.147Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:58:03.147Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:58:03.147Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:59:03.128Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:59:03.128Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:59:03.128Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:59:03.129Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-11-30T23:59:03.129Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:30:33.579Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:30:33.579Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:30:33.580Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:30:33.580Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:30:33.580Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:31:33.588Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:31:33.591Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:31:33.592Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:31:33.592Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:31:33.592Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:44:33.503Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:44:33.503Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:44:33.503Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:44:33.504Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:44:33.504Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:45:33.545Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:45:33.545Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:45:33.545Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:45:33.545Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:45:33.545Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:46:33.534Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:46:33.534Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:46:33.534Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:46:33.535Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:46:33.535Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:47:33.594Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:47:33.594Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:47:33.595Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:47:33.595Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:47:33.595Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:50:33.542Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:50:33.542Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:50:33.542Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:50:33.543Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:50:33.543Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:51:33.592Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:51:33.593Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:51:33.593Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:51:33.593Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:51:33.593Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:52:33.556Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:52:33.556Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:52:33.556Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:52:33.557Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:52:33.557Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:53:33.604Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:53:33.604Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:53:33.604Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:53:33.604Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:53:33.604Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:58:05.522Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:58:05.523Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:58:05.523Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:58:05.523Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T00:58:05.524Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:22:56.691Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:22:56.691Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:22:56.691Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:22:56.691Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:22:56.691Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:24:56.684Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:24:56.684Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:24:56.684Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:24:56.684Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:24:56.684Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:27:56.726Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:27:56.727Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:27:56.727Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:27:56.727Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:27:56.727Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:28:56.700Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:28:56.700Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:28:56.700Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:28:56.701Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:28:56.701Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:40:46.626Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:40:46.627Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:40:46.627Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:40:46.627Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:40:46.627Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:42:02.149Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:42:02.150Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:42:02.150Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:42:02.150Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:42:02.150Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:43:01.830Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:43:01.831Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:43:01.831Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:43:01.831Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:43:01.831Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:45:01.762Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:45:01.762Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:45:01.762Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:45:01.763Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:45:01.763Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:47:01.784Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:47:01.785Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:47:01.785Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:47:01.785Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:47:01.785Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:57:01.846Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:57:01.846Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:57:01.846Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:57:01.846Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:57:01.846Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:58:01.862Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:58:01.862Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:58:01.862Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:58:01.862Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T01:58:01.862Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:03:01.863Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:03:01.864Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:03:01.864Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:03:01.864Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:03:01.864Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.146","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.146:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.146:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:10:01.924Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:10:01.924Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:10:01.924Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:10:01.924Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:10:01.924Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"162.247.242.27","port":443,"stack":"Error: connect ETIMEDOUT 162.247.242.27:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 162.247.242.27:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:16:01.970Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:16:01.971Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:16:01.971Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:16:01.971Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:16:01.971Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:20:01.961Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:20:01.961Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:20:01.961Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:20:01.961Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:20:01.961Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:23:02.014Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:23:02.015Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:23:02.015Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:23:02.015Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:23:02.015Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:25:02.023Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:25:02.023Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:25:02.023Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:25:02.023Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:25:02.024Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:30:02.134Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:30:02.135Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:30:02.135Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:30:02.135Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":2319,"time":"2018-12-01T02:30:02.135Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:31:35.847Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:31:36.216Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:33:09.982Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:33:09.982Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxMjI2MDQ4LGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDUxNTU0MTIzNl0.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:36:26.554Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:36:26.555Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:36:26.556Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:36:26.557Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:36:26.557Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:37:26.528Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:37:26.528Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:37:26.528Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:37:26.529Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:37:26.529Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:40:26.467Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:40:26.467Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:40:26.467Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:40:26.467Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:40:26.467Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:43:26.470Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:43:26.471Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:43:26.471Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:43:26.471Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:43:26.471Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:44:26.475Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:44:26.475Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:44:26.476Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:44:26.476Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:44:26.476Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:46:26.597Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:46:26.597Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:46:26.598Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:46:26.598Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:46:26.598Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:47:26.598Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:47:26.598Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:47:26.598Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:47:26.599Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:47:26.599Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:49:26.589Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:49:26.590Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:49:26.590Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:49:26.590Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:49:26.590Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:51:26.584Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:51:26.584Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:51:26.584Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:51:26.584Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:51:26.584Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:56:26.699Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:56:26.699Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:56:26.699Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:56:26.699Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:56:26.699Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:57:26.685Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:57:26.686Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:57:26.686Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:57:26.686Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:57:26.686Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:59:26.535Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:59:26.535Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:59:26.535Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:59:26.535Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T02:59:26.535Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:01:26.637Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:01:26.638Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:01:26.638Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:01:26.638Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:01:26.638Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:02:26.674Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:02:26.674Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:02:26.674Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:02:26.674Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:02:26.674Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:03:26.679Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:03:26.679Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:03:26.679Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:03:26.680Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:03:26.680Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:04:26.666Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:04:26.666Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:04:26.666Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:04:26.667Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:04:26.667Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:16:26.765Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:16:26.765Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:16:26.765Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:16:26.765Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":14996,"time":"2018-12-01T03:16:26.765Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15174,"time":"2018-12-01T03:18:35.771Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15174,"time":"2018-12-01T03:18:36.104Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:19:04.968Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:19:05.172Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:19:07.236Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:19:07.237Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxMzkwMjk2LGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDIyNzgxODgxNTdd.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:21:23.733Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:21:23.735Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:21:23.735Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:21:23.736Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15181,"time":"2018-12-01T03:21:23.736Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15199,"time":"2018-12-01T03:21:32.527Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15199,"time":"2018-12-01T03:21:32.719Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15206,"time":"2018-12-01T03:22:04.272Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15206,"time":"2018-12-01T03:22:04.471Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15211,"time":"2018-12-01T03:22:15.913Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15211,"time":"2018-12-01T03:22:16.102Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15211,"time":"2018-12-01T03:22:17.992Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15211,"time":"2018-12-01T03:22:17.993Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxMzk4NzM5LGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDQyNDMyNDIyN10.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15224,"time":"2018-12-01T03:22:26.784Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15224,"time":"2018-12-01T03:22:26.983Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15239,"time":"2018-12-01T03:23:14.599Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15239,"time":"2018-12-01T03:23:14.802Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15253,"time":"2018-12-01T03:23:50.290Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15253,"time":"2018-12-01T03:23:50.495Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15253,"time":"2018-12-01T03:23:52.372Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15253,"time":"2018-12-01T03:23:52.373Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxNDAyNDUwLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDMzMzI0OTk1MV0.","component":"collector_api"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:24:38.742Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:24:38.938Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:29:29.053Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:29:29.053Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxNDE2MzYxLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDIyMDI5OTI4MDVd.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:31:45.505Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.149","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.149:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.149:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:31:45.508Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:31:45.508Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:31:45.508Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:31:45.508Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.149","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.149:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.149:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:33:45.595Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:33:45.596Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:33:45.596Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:33:45.596Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:33:45.596Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:38:45.723Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:38:45.724Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:38:45.724Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:38:45.724Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15269,"time":"2018-12-01T03:38:45.724Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15386,"time":"2018-12-01T03:39:46.783Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15386,"time":"2018-12-01T03:39:47.109Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:42:50.000Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:42:50.209Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:42:52.088Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:42:52.089Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxNDYxMjQ1LGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDE0NDI1NDA4ODld.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:45:08.655Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:45:08.656Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:45:08.656Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:45:08.657Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:45:08.657Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:49:08.638Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:49:08.638Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:49:08.638Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:49:08.639Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:49:08.639Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.145","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.145:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.145:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:50:08.720Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:50:08.720Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:50:08.721Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:50:08.721Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:50:08.721Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:51:08.836Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:51:08.837Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:51:08.837Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:51:08.837Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15403,"time":"2018-12-01T03:51:08.837Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T03:54:59.040Z","msg":"Using New Relic for Node.js. Agent version: 4.11.0; Node version: v10.11.0."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T03:54:59.373Z","msg":"Starting New Relic for Node.js connection process."}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T03:56:31.937Z","msg":"Reporting to: https://rpm.newrelic.com/accounts/2172716/applications/91367319","component":"new_relic_response"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T03:56:31.938Z","msg":"Connected to collector-005.newrelic.com:443 with agent run ID WzIse2E6MjcxNDk2MDYxLGI6OTEzNjczMjAsYzoyMTcyNzE2LGQ6IjQuMTEuMCIsZToibm9kZWpzIixmOiJSYW9zLU1hY0Jvb2stUHJvLmxvY2FsIixnOlt7YTo5MTM2NzMxOSxiOiJNeSBfcmdfcHJveHkifV19LDI4NDMxNzA0OTFd.","component":"collector_api"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:00:48.823Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:00:48.824Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:00:48.824Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:00:48.825Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:00:48.825Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:03:48.507Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:03:48.508Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:03:48.508Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:03:48.508Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:03:48.508Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":50,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:04:48.495Z","msg":"Calling metric_data on New Relic failed unexpectedly. Data will be held until it can be submitted:","component":"collector_api","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:04:48.495Z","msg":"Merging 0 (of 0) errors for next delivery.","component":"error_tracer"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:04:48.496Z","msg":"Merging 0 of 0 events into ErrorAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":40,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:04:48.496Z","msg":"Merging 0 of 0 events into SpanAggregator for next harvest","component":"event_aggregator"}
{"v":0,"level":30,"name":"newrelic","hostname":"Raos-MacBook-Pro.local","pid":15490,"time":"2018-12-01T04:04:48.496Z","msg":"Error on submission to New Relic (data held for redelivery):","errno":"ETIMEDOUT","code":"ETIMEDOUT","syscall":"connect","address":"50.31.164.148","port":443,"stack":"Error: connect ETIMEDOUT 50.31.164.148:443\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)","message":"connect ETIMEDOUT 50.31.164.148:443"}
